js/src/routes.js

```diff
@@ -60,6 +62,7 @@ const contractsRoutes = [
 ];
 
 const settingsRoutes = [
+  { path: 'advanced', component: SettingsAdvanced },
```

Comment:
Well I thought this would be more like a hidden feature for very specific edge cases which is debugging unconfirmed issues. Thus not showing the feature, but still easily accessible if needed.